V1

Oba-One opened the pull request.

mrhouzlane committed:
v1 - init system updated

mrhouzlane committed:
fix : v1 set params

mrhouzlane committed:
v1 MoveSystem added + libraries

mrhouzlane committed:
test v0

mrhouzlane committed:
forge install: forge-std

v1.3.0

mrhouzlane committed:
add

Oba-One committed:
dummy commit

junaama commented on packages/contracts/src/components/PhaseComponent.sol:
```diff
@@ -2,18 +2,18 @@
 
-contract PositionComponent is Uint256Component {
-  constructor(address world) Uint256Component(world, ID) {}
+contract PhaseComponent is Uint256Component {
```
why is phase replacing position?

Oba-One replied:
typo

junaama commented on packages/contracts/src/components/ZoneComponent.sol:
```diff
@@ -4,9 +4,9 @@ pragma solidity >=0.8.0;
+uint256 constant ID = uint256(keccak256("component.Zone"));
 
-contract PositionComponent is Uint256Component {
+contract ZoneComponent is Uint256Component {
```
why is zone replacing position?

Oba-One replied:
zone is not replacing position I copied the position component but forgot to change the name to zone. Zone refers to the 4 spots on a location and is tied to the character.

junaama commented on packages/contracts/src/libraries/LibCardMove.sol:
```diff
@@ -16,6 +16,4 @@ import { Uint32Component } from "std-contracts/components/Uint32Component.sol";
-library LibCardMove {
-
-}
+library LibCardMove {}
```
unimplemented

junaama reviewed (commented):
most of the logic still isn't there @mrhouzlane also we need more tests to ensure data is being updated properly

marvinmarnold committed:
build: sync React and Unity for web3 connection (#16)

* feat(unity): added setup screens

* fix(unity): empty contract params and react input

- Allow React input when Unity game rendered: https://docs.unity3d.com/ScriptReference/WebGLInput-captureAllKeyboardInput.html
- Support empty params with sendContract
- React hardcoded to MUD testnet

* fixes

marvinmarnold committed:
feat(react): point app to MUD testnet by default (#17)

mrhouzlane committed:
feat : added systems + PrevAct Components
